pod/perlstyle.pod

```diff
@@ -16,6 +16,13 @@ disable.  The B<-w> flag and C<$^W> variable should not be used for this
 purpose since they can affect code you use but did not write, such as
 modules from core or CPAN.
 
+A short and concise way to arrange for this is to use the
+L<C<use VERSION>|perlfunc/use VERSION> syntax, requesting a version 5.36
+or above, which will enable both the C<strict> and C<warnings> pragmata (as
+well as several other useful named features).
```

Comment:
This should link to `feature`, like `strict` and `warnings` do in the preceding paragraph.